src/server.js

```diff
@@ -206,7 +211,7 @@ function reverseDnsLookup(socket, client, token) {
 			client.hostname = client.ip;
 		}
 
-		init(socket, client, token);
+		init(socket, client, remember);
```

Comment:
Would need to store it somewhere while auth is happening (or disable the checkbox). I'd rather keep it as it is for now.